galleries/javed/index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <title></title>
+
+  </head>
+  <body>
+    <div class="galleri">
+      <h1>DaylyGalleri</h1>
+      <div class="pics">
+        <div class="pic">
+          <div class="img">
+            <img src="bil.jpg" alt="">
+          </div>
+          <div class="discrip">
+            <div class="likes">
+              <i class="material-icons">favorite</i>
+              <span>234 Likes</span>
+            </div>
+            <div class="text">
+              <p>this is a description about the image</p>
+            </div>
+          </div>
+        </div>
+        <div class="pic">
+          <div class="img">
```

Comment:
I think one wrapper div is enough for the image. I am confused about what is the difference between .pic and .img

Reply:
sorry about bad name choices, but I think it needs. pic is for whole container and "img" is just for image. I think it needs to make grid work.
I made changes, if it make sense now.